frontend/components/forms/FormImageUpload.tsx

```diff
@@ -0,0 +1,65 @@
+import ImageWithOverlay from '../generic/ImageWithOverlay';
+import Button from '../generic/Button';
+import AddPhotoAlternateIcon from '@mui/icons-material/AddPhotoAlternate';
+import { useField } from 'formik';
+import { ChangeEvent } from 'react';
+import { Stack } from '@mui/system';
+import DeleteIcon from '@mui/icons-material/Delete';
+import { Typography } from '@mui/material';
+
+interface Props {
+  name: string;
+}
+
+const FormImageUpload = ({ name }: Props) => {
```

Comment:
Is there a way to validate the size of the image. We only allow max 1MB per image in the backend.